### PR TITLE
Feature/aironet 1600

### PR DESCRIPTION
**Pull request by whitx**
# Description

Cisco Aironet 1600 support, for deauth/auth radius.
# Impacts

Switch module.
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Module for Cisco Aironet 1600.

### CODE REVIEW

File: NEWS.asciidoc [hunk 1]
```diff
@@ -24,6 +24,7 @@ New Features
 * New api method for adding a person
 * New api method for viewing a person
 * New api method for modifying a person
+* Added Cisco aironet 1600 switch module
```
Comment (extrafu) [inline NEWS.asciidoc]: proper capitalization -> Aironet

**Comment by fdurand** [comment]
If you can update PacketFence_Network_Guide with a sample config for the AP

**Comment by extrafu** [comment]
@fdurand Not a can, a _must_ ! Good point.

File: lib/pf/Switch/Cisco/Aironet_1600.pm [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+=cut
+
+use strict;
+use warnings;
+use Log::Log4perl;
+use Net::SNMP;
+
+use pf::config;
+use pf::Switch::constants;
+use pf::util;
```
Comment (fdurand) [inline lib/pf/Switch/Cisco/Aironet_1600.pm]: Useless

File: lib/pf/Switch/Cisco/Aironet_1600.pm [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+Send a CoA to disconnect a mac
+
+=cut
+
+sub radiusDisconnect {
+    my ($self, $mac, $add_attributes_ref) = @_;
+    my $logger = Log::Log4perl::get_logger( ref($self) );
+
+    $logger->warn($mac);
```
Comment (fdurand) [inline lib/pf/Switch/Cisco/Aironet_1600.pm]: Remove

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 4]
```diff
@@ -3298,6 +3298,19 @@ aaa group server radius rad_mac
...
 
+Aironet 1600
+^^^^^^^^^^^^
+
+CoA and radius:
+----
+radius-server attribute 32 include-in-access-req format %h
+radius-server vsa send accounting
+aaa server radius dynamic-author
+ client 192.168.0.10
```
Comment (dwlfrth) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: Is that supposed to be the PacketFence server ? If yes, the guide assumption is '192.168.1.5'

**Comment by dwlfrth** [comment]
Is it working ?
It does compile, it passes smoke tests, and code looks ok.
Since it is fairly some copy pasted code, I assume it basically works but, is it working for the intended purpose ?

**Comment by extrafu** [comment]
AFAIK, it works at a customer site.

**Pull request merged**